src/open_inwoner/templates/pages/profile/contacts/list.html

```diff
@@ -116,7 +116,9 @@ <h2 class="h2">{% trans "U bent toegevoegd als contactpersoon" %}</h2>
                         <td class="table__item">{{contact.email}}</td>
                         <td class="table__item"></td>
                         <td class="table__item">{% icon "close" extra_classes="icon icon--danger" %}</td>
-                        <td class="table__item">{% button text=_('Stuur bericht') icon="message" icon_position="before" icon_outlined=True transparent=True disabled=True %}</td>
+			{% if not messages_enabled %}
```

Comment:
I think it doesn't make sense to show a disabled button if the functionality is also disabled (for both the accepted and not users). It's not something that the user can have even if his invitation is accepted. @alextreme can tell us his opinion as well.

If we do show the disabled button, we have to fix the styling.

![image](https://github.com/maykinmedia/open-inwoner/assets/63058071/4b1e1089-d74d-4760-bcb1-4696ae7ec64b)

Reply:
Agreed, I think I mentioned earlier to not show this button if mijn berichten is disabled